docs/API-REFERENCE.md

```diff
@@ -1 +1,2640 @@
+
+  ... // usage of the value
+
+  jerry_release_value (number_value);
+}
+```
+
+**See also**
+
+- [jerry_release_value](#jerry_release_value)
+
+
+# jerry_create_object_value
+
+**Summary**
+
+Converts the given `jerry_object_t` parameter to a `jerry_value_t`. The value has to be released
+by [jerry_release_value](#jerry_release_value) when it becomes unnecessary.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_create_object_value (jerry_object_t *value);
+```
+
+- `value` - `jerry_object_t` from which a `jerry_value_t` will be created.
+- return value - a `jerry_value_t` created from the given `jerry_object_t` argument.
+
+**Example**
+
+```c
+{
+  jerry_init (JERRY_FLAG_EMPTY);
+  jerry_object_t *object_p = jerry_create_object ();
+  jerry_value_t object_value = jerry_create_object_value (object_p);
+
+  ... // usage of object_value
+
+  jerry_release_value (object_value);
+}
+```
+
+**See also**
+
+- [jerry_release_value](#jerry_release_value)
+
+
+# jerry_create_string_sz
+
+**Summary**
+
+Creates a string. The caller should release the string with [jerry_release_string](#jerry_release_string)
+when it becomes unnecessary.
+
+**Prototype**
+
+```c
+jerry_string_t *
+jerry_create_string_sz (const jerry_char_t *str_p,
+                        jerry_size_t str_size)
+```
+
+- `str_p` - string from which the result `jerry_string_t` will be created.
+- `str_size` - size of the string.
+- return value - a `jerry_string_t*` created from the given `jerry_char_t*` argument.
+
+**Example**
+
+```c
+{
+  jerry_init (JERRY_FLAG_EMPTY);
+  const jerry_char_t char_array[] = "a string";
+  jerry_string_t *string  = jerry_create_string_sz (char_array,
+                                                    strlen ((char *) char_array));
+
+  ... // usage of string
+
+  jerry_release_string (string);
+}
+
+```
+
+**See also**
+
+- [jerry_release_string](#jerry_release_string)
+
+
+# jerry_create_string_value
+
+**Summary**
+
+Creates a `jerry_value_t` from the given `jerry_string_t` parameter. The value has to be released
+by [jerry_release_value](#jerry_release_value) when it becomes unnecessary.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_create_string_value (jerry_string_t *value);
+```
+
+- `value` - `jerry_string_t` from which a value will be created.
+- return value - a `jerry_value_t` created from the given `jerry_string_t` argument.
+
+**Example**
+
+```c
+{
+  jerry_init (JERRY_FLAG_EMPTY);
+  const jerry_char_t char_array[] = "a string";
+  jerry_string_t *string  = jerry_create_string_sz (char_array,
+                                                    strlen ((char *) char_array));
+  jerry_value_t string_value = jerry_create_string_value (string);
+
+  ... // usage of string_value
+
+  jerry_release_value (string_value);
+}
+```
+
+**See also**
+
+- [jerry_release_value](#jerry_release_value)
+
+
+# jerry_create_error_sz
+
+**Summary**
+
+Creates an error object. Caller should release the object with jerry_release_object, just when
```

Comment:
There are multiple instances of `just when ...`, please change these to simply say `when ...`